Library/ViewModels/ProjectPamphletMainCellViewModel.swift

```diff
@@ -328,7 +328,10 @@ public final class ProjectPamphletMainCellViewModel: ProjectPamphletMainCellView
 
     let shouldTrackCTATappedEvent = projectAndRefTag
       .takeWhen(self.readMoreButtonTappedProperty.signal)
-      .filter { project, _ in project.state == .live && project.personalization.isBacking == false }
+      .logEvents(identifier: "***")
+      .filter { project, _ in
+        project.state == .live && project.personalization.backing == nil
```

Comment:
Ahhh this is such a common pitfall for us. Do you think it could be worthwhile to add a helper function to `SharedFunctions.swift` to handle all the logical cases for this? Something like

```
func currentUserHasBacked(project: Project) {
    return project.personalization.backing != nil || project.personalization.isBacking == .some(true)
}
```
To be honest I'm not entirely sure that `personalization.isBacking` means what we think it does even though we use it all over the place 🤔.

Reply:
Sure I could do that. It looks like I also missed a `logEvents` still in the code there 😂